src/fragments/lib/troubleshooting/common/upgrading.mdx

```diff
@@ -117,7 +117,7 @@ Note that v6 supports react-native v0.70+, so if you prefer manually upgrading d
 
 ```diff
 - platform :ios, min_ios_version_supported
-+ platform :ios, 13.0
++ platform :ios, 13.4
```

Comment:
Please hold, discussing if we should remove this info entry from the doc.

Reply:
Discussed offline - will be removing this entirely.